docs/pages/admin-guides/access-controls/access-graph/ssh-keys-scan.mdx

```diff
@@ -0,0 +1,196 @@
+
+### Importing User's SSH Private Keys
+
+Teleport's `tsh` CLI tool can scan users' laptops for SSH private keys.
+It goes through the specified directories—defaulting to `/Users` on macOS, `/home` on Linux, and `C:\Users` on Windows—
+by examining files with certain prefaces to identify SSH private keys.
+
+The `tsh` tool authenticates with the Teleport Cluster through the [Device Trust](../device-trust.mdx) feature,
+which guarantees that only enrolled devices can submit private keys to the cluster. By utilizing the device's Secure
+Enclave or TPM private key, it confirms that the device is the same one that was enrolled, enabling Teleport to trust
+and accept the private key reports without requiring further authentication or credentials thus allowing scanning operations
+to be performed without user interaction and valid credentials.
+
+Once the device is authenticated, the `tsh` tool extracts the public key fingerprint from the private key, if available,
+otherwise, it generates the fingerprint from the public key file if the last exists. The fingerprint is then sent to the
+Teleport Cluster with metadata such as:
+
+- `device_id`: The unique identifier of the device.
+- `public_fingerprint`: The SHA256 fingerprint of the public key if available.
+- `derivation_mode`: The method used to derive the public key from the private key. If can be either `private_key` if the
+  public key is extracted from the private key, `public_key_file` if the private key is encrypted and the public key is
+    available in a separate file, or `protected` if the public key fingerprint is not available.
+
+<Notice type="warning">
+`tsh` never sends the private key itself to the Teleport Cluster, ensuring that the private key remains secure on the device.
+It also never sends the private key path or any other sensitive information.
+</Notice>
+
+## Prerequisites
+
+- A running Teleport Enterprise cluster v15.3.X/v16.2.0 or later.
+- For self-hosted clusters, an updated `license.pem` with Teleport Policy enabled.
+- For self-hosted clusters, a running Access Graph node v1.22.0 or later.
+Check [Access Graph page](../teleport-policy/getting-started-policy.mdx) for details on
+how to set up Access Graph.
+- For self-hosted clusters, the node running the Access Graph service must be reachable
+from Teleport Auth Service.
+- A Teleport Node running the Teleport SSH Service.
+- Devices enrolled in the [Teleport Device Trust feature](../device-trust.mdx).
+- For Jamf Pro integration, devices must be enrolled in Jamf Pro and have the signed `tsh` binary installed.
+
+## Step 1/3. Enable SSH Key Scanning
+
+To enable SSH Key Scanning, you need to configure the Teleport cluster to scan for SSH Authorized Keys.
+
+To enable the SSH Key Scanning feature, edit the Teleport Access Graph configuration file and set the
+`spec.secrets_scan_config` field to `enabled` as shown below:
+
+```bash
+$ tctl edit access_graph_settings
```

Comment:
Are we planning to have an option to enable this in the UI?

Reply:
I didn't have plans to do it mostly because all config options are only available via tctl like changing the cluster_auth_preference. We don't offer an option to edit it via UI.
Should I add it?